app/web/package.json

```diff
@@ -22,41 +22,41 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@date-io/dayjs": "1.x",
-    "@emotion/cache": "^11.13.1",
     "@emotion/react": "^11.13.5",
-    "@emotion/server": "^11.11.0",
     "@emotion/styled": "^11.13.5",
     "@fontsource/mansalva": "^5.1.0",
     "@fontsource/ubuntu": "^5.1.0",
     "@mui/icons-material": "^5.16.8",
     "@mui/lab": "^5.0.0-alpha.164",
     "@mui/material": "^5.16.7",
     "@mui/styles": "^5.16.7",
+    "@mui/system": "^5.4.1",
     "@mui/utils": "^5.16.6",
     "@mui/x-date-pickers": "^5.0.20",
     "@sentry/nextjs": "^6.19.7",
     "@sentry/react": "^8.41.0",
     "@stripe/stripe-js": "^5.2.0",
     "@toast-ui/editor": "^3.2.2",
-    "@types/react-simple-maps": "^3.0.6",
     "classnames": "^2.5.1",
     "dayjs": "^1.11.13",
     "frontmatter-markdown-loader": "^3.7.0",
+    "geojson": "^0.5.0",
     "glob": "^8.0.3",
     "google-protobuf": "^3.21.4",
     "grpc-web": "1.5.0",
+    "i18next": "^21.5.3",
     "intersection-observer": "^0.12.2",
     "luhn": "^2.4.1",
     "maplibre-gl": "^4.7.1",
     "markdown-it": "^14.1.0",
     "next": "12.1.6",
     "next-i18next": "git+https://github.com/Couchers-org/next-i18next.git#39a50ec9",
```

Comment:
Probably one for a separate task... but I reckon we can probably move off this forked version of `next-i18next` so we are receiving updates again. I believe it was pinned to this as Lucas (one of the former active maintainers alongside with me!) made a fix upstream and it was unclear when they were merging and making a proper release with the fix. I believe the fix has been accepted now though so probably safe to go back to an official version instead.

Reply:
Can confirm it was taken into upstream here! https://github.com/i18next/next-i18next/pull/1706

Reply:
Ah amazing I actually recently tagged Lucas in the PR he did this in asking the story behind it but even better that you know! That was on my radar to deal with eventually so I'm happy someone remembers why it was done so I don't have to stab in the dark!